app/uk/gov/hmrc/helptosavefrontend/config/FrontendWiring.scala

```diff
@@ -109,4 +109,18 @@ class WSHttpProxy extends WSPost with WSProxy with RunMode with HttpAuditing wit
     executeHooks(url, POST_VERB, None, httpResponse)
     httpResponse
   }
+
+  /**
+   * Returns a [[Future[HttpResponse]] without throwing exceptions if the status us not `2xx`. Needed
+   * to replace [[PUT]] method provided by the hmrc library which will throw exceptions in such cases.
+   */
+  def put[A](url:     String,
+             body:    A,
+             headers: Map[String, String] = Map.empty[String, String]
+  )(implicit rds: Writes[A], hc: HeaderCarrier, ec: ExecutionContext): Future[HttpResponse] = {
+    // cannot use `doPut` over here because it doesn't allow for headers
+    val httpResponse = buildRequest(url).withHeaders(headers.toList: _*).put(Json.toJson(body)).map(new WSHttpResponse(_))
+    executeHooks(url, POST_VERB, None, httpResponse)
```

Comment:
is this meant to be `PUT_VERB`  or `POST_VERB` ?

Reply:
yes, whoops